testsuite/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.google.devtools.ksp") version "1.9.20-1.0.13"
+}
+
+kotlin {
+    // targets have to be the same as in :cli (because it depends on this subproject)
+    linuxX64()
+//    linuxArm64()
```

Comment:
```suggestion
    // linuxArm64()
```

Reply:
but this is what IDEA does when I press `ctrl+/`...

Reply:
You can manage this behavior in the settings

![Screenshot from 2024-02-05 12-22-30](https://github.com/JetBrains-Research/litmuskt/assets/3542287/c783c29f-451a-4582-8d17-e42fcc76a62f)